.github/workflows/python-package.yml

```diff
@@ -21,7 +21,9 @@ jobs:
           python-version: ${{ matrix.python-version }}
           cache: "poetry"
       - run: poetry install
-      - run: poetry run pytest --cloud-api-key ${{ secrets.CLOUD_API_KEY }}
+      - run: poetry run pytest
+      - run: poetry run pytest -m cloud --cloud-api-key "${{ secrets.CLOUD_API_KEY }}"
+      - run: poetry run pytest -m enterprise --enterprise-api-key "${{ secrets.ENTERPRISE_API_KEY }}"
```

Comment:
I think we should also specify the URL as a parameter instead of hard-coding dev in the code, so there's only one place to change it and so developers can test against different clusters (e.g. a customer could even run the tests against their own version). In particular, I wonder if we should use test instead of dev.

Reply:
Should we do the same for the cloud tests?

Also, I chose dev because that is what we are using for our E2E tests.

Reply:
I think SystemLink Cloud is different, because it only exists at systemlinkcloud.io, though there are technically multiple tiers as well.

The E2E tests are testing the product itself, which makes sense to do at each tier, while these tests are for the clients. In the future, we could add an E2E test that pulls and runs these tests against each tier as well, to detect API breaks.